src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/HostingDiagnosticListener.cs

```diff
@@ -99,6 +150,124 @@ public void OnDiagnosticsUnhandledException(HttpContext httpContext, Exception e
             this.OnException(httpContext, exception);
         }
 
+        private RequestTelemetry InitializeRequestTelemetryOnHttpRequestInStart(HttpContext httpContext)
+        {
+            var requestTelemetry = new RequestTelemetry();
+
+            if (Activity.Current == null)
+            {
+                AspNetCoreEventSource.Instance.LogHostingDiagnosticListenerOnHttpRequestInStartActivityNull();
```

Comment:
this means something really strange happened and AspNEtCore did not create activity for us but sent the event. return, there is nothing else we can do